Zype/Managers/AnalyticsManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  AnalyticsManager.swift
+//  Zype
+//
+//  Created by Andy Zheng on 1/24/18.
+//  Copyright © 2018 Eugene Lizhnyk. All rights reserved.
+//
+
+import Foundation
+
+class AnalyticsManager: NSObject {
+    // MARK: - Properties
+    open static let sharedInstance = AnalyticsManager()
+
+    // MARK: - reset
+    open func reset() {
+        AKAMMediaAnalytics_Av.deinitMASDK()
+    }
+    
+    // MARK: - track video playback
+    open func trackPlay(withConfigUrl url: URL, withPlayer player: AVPlayer, withCustomData data: Dictionary<String, String>){
```

Comment:
trackPlay would not be a reusable function since you are passing player to the framework which not just tracking a single play event.